components/automate-ui/src/app/page-components/projects-filter-dropdown/projects-filter-dropdown.component.scss

```diff
@@ -99,6 +99,7 @@
 
     .button-container { // adjust for 8px global margin on buttons
       display: flex;
+      justify-content: space-between;
```

Comment:
The two buttons are no longer static; the width of the "Clear Selection" button can vary. Nonetheless, this keeps them pushed to their respective left/right edges, to maintain necessary alignment with other components.

Reply:
would it be easy to keep the buttons 16px apart instead? so the space to the right of clear button will vary instead of the space between the buttons?

Reply:
That was my first attempt, but it looks askew when you approach the button (i.e. hover over it).
Here it aligns-right with the form field above:
![image](https://user-images.githubusercontent.com/6817500/78930127-6e08c280-7a58-11ea-9129-863859434fe8.png)
vs. this:
![image](https://user-images.githubusercontent.com/6817500/78930408-ec656480-7a58-11ea-885c-ccd0e2d8c938.png)

Reply:
the second image is what i want.

Reply:
its more important that the beginning of the button does not move, as that can often be a frustrating experience. im unconcerned about alignment with the form input.

Reply:
Changed.